### PR TITLE
[DX-1609] Update long-term-support-releases.md

### PR DESCRIPTION
**Pull request by letzya**
### For internal users - Please add a Jira DX PR ticket to the subject!
[DX-1609](https://tyktech.atlassian.net/browse/DX-1609)

---

### Preview Link

[preview](https://deploy-preview-5212--tyk-docs.netlify.app/docs/nightly/developer-support/special-releases-and-features/long-term-support-releases)

### Description
Make the "maintenance support" clearer, however, I think we need to have a more detailed explanation.

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

[DX-1609]: https://tyktech.atlassian.net/browse/DX-1609?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dcs3spp — APPROVED** [review]
PR LGTM, will merge and release once peer approved

**Comment by letzya** [comment]
@dcs3spp what's the blocker on this one? why do we need peer approval on top of yours for this PR?

File: tyk-docs/content/developer-support/special-releases-and-features/long-term-support-releases.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Our current long term support release is 5.3 LTS. This is in full support from M
 
 ## What Is Our LTS Offering?
 
-We provide full support for the first 12 months, including regular maintenance intervals every seven weeks. Following this period, we transition to maintenance support for an additional 12 months, focusing on critical fixes and essential security patching as needed.
+We provide **full support for the first 12 months**, including regular maintenance intervals every seven weeks. Following this period, we transition to **maintenance support for an additional 12 months**, focusing on critical fixes and essential security patching as needed.
```
Comment (letzya) [inline tyk-docs/content/developer-support/special-releases-and-features/long-term-support-releases.md:32]: @hagartyk dvlffdlk

Comment (letzya) [inline tyk-docs/content/developer-support/special-releases-and-features/long-term-support-releases.md:32]:
```suggestion
We provide **full support for the first 12 months**, including regular maintenance intervals every seven weeks. Following this period, we transition to bla bla **maintenance support for an additional 12 months**, focusing on critical fixes and essential security patching as needed.
```

**Review by allaneyles — COMMENTED** [review]
All good @letzya

**Pull request merged**